client/src/components/ACL/AclPage.js

```diff
@@ -62,11 +62,10 @@ const groupColumns = [
         sortable: true,
     },
     {
-        key: "acl",
-        name: "Predicates",
+        key: "totalPredicateCount",
```

Comment:
Why is this being changed? Is it backward compatible?

Reply:
Instead of showing predicates list, predicatesCount will be shown now.

Discussed this with @anand-chandrashekar